clients/spark/core/src/main/scala/io/treeverse/clients/StorageUtils.scala

```diff
@@ -1,11 +1,24 @@
 package io.treeverse.clients
 
 import java.net.URI
+import java.nio.charset.Charset
 
 object StorageUtils {
   val StorageTypeS3 = "s3"
   val StorageTypeAzure = "azure"
 
+  def concatKeysToStorageNamespace(
```

Comment:
It sounds like this should be an abstract method / interface method - and the other static objects in this file should be implementations.

Reply:
Unfortunately I don't have time to address this comment this week, will open an issue to track it :)